test/e2e/go.mod

```diff
@@ -0,0 +1,20 @@
+module yunikorn-qe
```

Comment:
looks like this is a separate module, I do not think go mod can support multiple modules per repo nicely. Based on the comment:

> For all but power users, you probably want to adopt the usual convention that one repo = one module. It's important for long-term evolution of code storage options that a repo can contain multiple modules, but it's almost certainly not something you want to do by default.

See more here: https://github.com/golang/go/wiki/Modules#should-i-have-multiple-modules-in-a-single-repository.